src/_stories/gis-update.md

```diff
@@ -0,0 +1,49 @@
+---
+schema: default
+title: Our geo data has become more relevant
+date: 2017-08-10 00:00:00
+thumbnail: /assets/img/stories/zoning-workflow.jpg
+author:
+    name: Arnaud Vedy
+---
+
+Dear readers  / followers, we have some exciting news!
+Our geographic data is now integrated with Poseidon, our data automation system.
+
+**What does this mean for you?**
+
+<!--more-->
+
+### Current geo data
+
+It means you now have access to the most current geographic data at all times. Some data sources are updated every month. For others, this process occurs on a daily basis. No matter the schedule of the dataset, our portal will now automatically update along with it. This ensures that as a user of the portal, you will be retrieving the most contemporary data available.
+![Data automation workflow for our zoning geo datasets](/assets/img/stories/zoning-workflow.jpg)
+
+### New formats
+
+Until recently, we were sharing geo data exclusively as shapefiles. This is one of the most commonly used format for geo data. Nevertheless, there are a few reasons why it should not be the only format of geo data offered on an open data portal:
+- It is not an open standard format.
+- It is actually multiple files. These files need to be zipped, making it inconvenient to share.
+- It is not a practical format for web developers. Few web mapping javascript libraries support shapefiles natively, which usually leads to data conversion.
+- It is not the most practical format for non-technical users. It requires software installation to be visualized / used for data analysis.
+For these reasons, we will be sharing our geo data in other formats from now on. The formats available for now are GeoJSON, TopoJSON, and Geobuf.
```

Comment:
More clarification -> 

For these reasons, we will be additional geodata formats. For now (and we will be adding more) they are: GeoJSON, TopoJSON, and Geobuf.